### PR TITLE
Remove author

### PR DESCRIPTION
**Pull request by OskarStark**
I am targeting this branch, because this is BC.

## Subject

Only some cleanup, **please review commit by commit**!

### CODE REVIEW

File: Resources/skeleton/odm/repository.mustache [hunk 1]
```diff
@@ -14,5 +14,4 @@ use {{ namespace}}\Document\Base{{ name }}Repository;
  */
 class {{ name }}Repository extends Base{{ name }}Repository
 {
-
 }
```
Comment (OskarStark) [inline Resources/skeleton/odm/repository.mustache]: not sure if we should change this to a newline in the templates

Comment (greg0ire) [inline Resources/skeleton/odm/repository.mustache]: Yes we should.

Comment (OskarStark) [inline Resources/skeleton/odm/repository.mustache]: ok, will do

Comment (OskarStark) [inline Resources/skeleton/odm/repository.mustache]: done

File: Resources/skeleton/serializer/document.mustache [hunk 2]
```diff
@@ -1,10 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <serializer>
-    <!--
-        This file has been generated by the EasyExtends bundle ( https://sonata-project.org/bundles/easy-extends )
-
-        @author <yourname> <youremail>
-    -->
+    <!-- This file has been generated by the SonataEasyExtendsBundle: https://sonata-project.org/bundles/easy-extends -->
```
Comment (core23) [inline Resources/skeleton/serializer/document.mustache]: Should we remove all newlines too, because `xmllint` would remove them when calling `make lint`.

Comment (OskarStark) [inline Resources/skeleton/serializer/document.mustache]: not sure

Comment (OskarStark) [inline Resources/skeleton/serializer/document.mustache]: done

**Comment by OskarStark** [comment]
@core23 I updated the files with the make lint command and now it fails:
https://travis-ci.org/sonata-project/SonataEasyExtendsBundle/jobs/234822642

Looks strange, shall I revert the commit then?

File: Resources/config/generator.xml [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-        <service id="sonata.easy_extends.generator.phpcr" class="Sonata\EasyExtendsBundle\Generator\PHPCRGenerator"/>
-        <service id="sonata.easy_extends.generator.serializer" class="Sonata\EasyExtendsBundle\Generator\SerializerGenerator"/>
-    </services>
+  <services>
+    <service id="sonata.easy_extends.generator.bundle" class="Sonata\EasyExtendsBundle\Generator\BundleGenerator"/>
+    <service id="sonata.easy_extends.generator.orm" class="Sonata\EasyExtendsBundle\Generator\OrmGenerator"/>
+    <service id="sonata.easy_extends.generator.odm" class="Sonata\EasyExtendsBundle\Generator\OdmGenerator"/>
+    <service id="sonata.easy_extends.generator.phpcr" class="Sonata\EasyExtendsBundle\Generator\PHPCRGenerator"/>
+    <service id="sonata.easy_extends.generator.serializer" class="Sonata\EasyExtendsBundle\Generator\SerializerGenerator"/>
+  </services>
```
Comment (greg0ire) [inline Resources/config/generator.xml]: We picked an indent of 4 spaces, that's why it fails.

Comment (OskarStark) [inline Resources/config/generator.xml]: but what is correct? shouldn't it execute the same command like `make lint` ?

Comment (core23) [inline Resources/config/generator.xml]: Ping @greg0ire

Comment (greg0ire) [inline Resources/config/generator.xml]: The make target should be fixed, [this](https://github.com/sonata-project/SonataEasyExtendsBundle/blob/553abfe6be5df697b703f082c9bd4c3b1ef79d8d/.travis.yml#L35) should be moved to the Makefile.

Comment (OskarStark) [inline Resources/config/generator.xml]: this should be done via devkit?

Comment (OskarStark) [inline Resources/config/generator.xml]: Editing the Makefile like this:
```Makefile
lint: export XMLLINT_INDENT="    "
lint:
	composer validate
	find . \( -name '*.xml' -or -name '*.xliff' \) \
		-not -path './vendor/*' -not -path './Resources/public/vendor/*' -type f \
		-exec xmllint --encode UTF-8 --output '{}' --format '{}' \;
	git diff --exit-code

```

results in:
```diff
-  <services>
-    <service id="sonata.easy_extends.doctrine.mapper" class="Sonata\EasyExtendsBundle\Mapper\DoctrineORMMapper">
-      <tag name="doctrine.event_subscriber"/>
-      <argument type="service" id="doctrine"/>
-      <argument type="collection"/>
-    </service>
-  </services>
+"    "<services>
+"    ""    "<service id="sonata.easy_extends.doctrine.mapper" class="Sonata\EasyExtendsBundle\Mapper\DoctrineORMMapper">
+"    ""    ""    "<tag name="doctrine.event_subscriber"/>
+"    ""    ""    "<argument type="service" id="doctrine"/>
+"    ""    ""    "<argument type="collection"/>
+"    ""    "</service>
+"    "</services>
```

removing the quotes or escaping them does not work either, any ideas?

Comment (OskarStark) [inline Resources/config/generator.xml]: This works: `XMLLINT_INDENT="    " make lint`

**Comment by OskarStark** [comment]
This one is RTM

**Comment by greg0ire** [comment]
Can you squash the xmllint commits together, or should we squash all this together?

**Comment by OskarStark** [comment]
> Can you squash the xmllint commits together, or should we squash all this together?

only the xml lint commits

**Comment by greg0ire** [comment]
Ok, but you know we can't do that for you, don't you?

**Comment by OskarStark** [comment]
done 😄

**Comment by greg0ire** [comment]
You still have 3 xml related commits :P

**Comment by OskarStark** [comment]
> You still have 3 xml related commits :P

oops..you were right 😄

**Comment by greg0ire** [comment]
Thanks @OskarStark !

**Pull request merged**